src/AlphaRPC/Console/Command/GenerateSupervisorConfig.php

```diff
@@ -78,9 +78,9 @@ public function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $group = $input->getArgument('group');
-        if (!preg_match('/^[A-Za-z0-9]+$/', $group)) {
+        if (!preg_match('/^[A-Za-z0-9]+(-[A-Za-z0-9]+)*$/', $group)) {
```

Comment:
Then '/^[\w-]+$/' would be even more clean, but this enables you to add dashes before and after the group name.